app/assets/stylesheets/application.scss

```diff
@@ -1,16 +1,6 @@
 @import "bootstrap";
-/*
- * This is a manifest file that'll be compiled into application.css, which will include all the files
- * listed below.
- *
- * Any CSS and SCSS file within this directory, lib/assets/stylesheets, or any plugin's
- * vendor/assets/stylesheets directory can be referenced here using a relative path.
- *
- * You're free to add application-wide styles to this file and they'll appear at the bottom of the
- * compiled file so the styles you add here take precedence over styles defined in any other CSS/SCSS
- * files in this directory. Styles in this file should be added after the last require_* statement.
- * It is generally better to create a new file per style scope.
- *
- *= require_tree .
- *= require_self
- */
+
+
+.service-container h3 {
```

Comment:
See the latest updates to the bootstrap setup - I think you'll want to put any css in a "management" directory in app/assets/stylesheets and import it above bootstrap in management.scss (something like that). It should not go in this file - as you can see from the comment in the previous version of this file.

Reply:
Addressed

Reply:
I think you need to revert this file to how it is in master, currently.